Gringotts-core/src/main/java/org/gestern/gringotts/api/impl/GringottsEco.java

```diff
@@ -16,7 +17,6 @@
 import java.util.Set;
 import java.util.UUID;
 
-import static org.gestern.gringotts.Configuration.CONF;
```

Comment:
why change the import?

Reply:
Probably my ide I suspect. I'll have another look. Normally try to change as little as possible in a pr but when messing around with the whole project to multi maven it, it can get lost in the changes.